pkg/runner/cluster_k8s.go

```diff
@@ -96,8 +97,21 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 		TestSidecar:        true,
 	}
 
+	var err error
+	_, runenv.TestSubnet, err = net.ParseCIDR("10.33.10.0/24")
```

Comment:
TODO: auto select a good network (ideally in the range 16. - 32. to match docker.

Reply:
That would involve changing the default Weave network. Will look into it.

Reply:
I've added it as a TODO in the PR description for follow up.